gnat2goto/driver/goto_utils.adb

```diff
@@ -154,52 +154,51 @@ package body GOTO_Utils is
       A_Symbol_Table.Insert (Subprog_Name, Subprog_Symbol);
    end New_Subprogram_Symbol_Entry;
 
-   procedure New_Type_Symbol_Entry (Type_Name : String; Type_Type : Irep;
+   procedure New_Type_Symbol_Entry (Type_Name : Symbol_Id; Type_Of_Type : Irep;
```

Comment:
That kind of makes it sound like we have higher kinded types? I honestly don't really have a fantastic idea of what to name this either though.